fastbril/README.md

```diff
@@ -0,0 +1,30 @@
+# The `fastbril` bytecode interpreter
+
+This is a bytecode spec/generator/interpreter for `bril`. It aims to
+be like the typescript or rust implementation, but faster.
+
+## To build
+
++ binary: `make release`
+  the binary will be `./build/fastbrili`
++ doc: you need to have LaTeX installed. run `make doc`
```

Comment:
FYI, I tried to run this and got:
```
Makefile:58: target `src/.h' given more than once in the same rule.
Makefile:58: target `src/.h' given more than once in the same rule.
Makefile:58: target `src/.h' given more than once in the same rule.
Makefile:58: target `src/.h' given more than once in the same rule.
Makefile:69: target `doc/.tex' given more than once in the same rule.
Makefile:69: target `doc/.tex' given more than once in the same rule.
Makefile:69: target `doc/.tex' given more than once in the same rule.
Makefile:69: target `doc/.tex' given more than once in the same rule.
mkdir -p doc/
./docgen.sh doc/.tex
awk: can't open file config/.cf
 source line number 0
make: *** [doc/.tex] Error 2
```

Reply:
this sounds like `find` isn't working for you the same way it does for me...